### PR TITLE
[fix](TVF) Enable azure tvf without specifying s3 bucket property.

### PR DESCRIPTION
**Pull request by ByteYue**
## Proposed changes

For azure blob storage, the region property is unnecessary. But the legacy tvf code would check if the properties contains bucket, so this pr tries to set one mock bucket property for azure tvf.

### CODE REVIEW

**Comment by ByteYue** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/tablefunction/S3TableValuedFunction.java [hunk 1]
```diff
@@ -57,6 +57,10 @@ public class S3TableValuedFunction extends ExternalFileTableValuedFunction {
 
     public S3TableValuedFunction(Map<String, String> properties) throws AnalysisException {
         final boolean isAzureTvf = AzureProperties.checkAzureProviderPropertyExist(properties);
+        // Azure could run without region
+        if (isAzureTvf) {
+            properties.put(S3Properties.REGION, "REGION");
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/tablefunction/S3TableValuedFunction.java:62]: "dummy-region"

**Comment by gavinchou** [comment]
Design needs to be review, pending merging

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**